skopt/plots.py

```diff
@@ -239,8 +256,8 @@ def partial_dependence(space, model, i, j=None, sample_points=None,
         return xi, yi, np.array(zi).T
 
 
-def plot_objective(result, levels=10, n_points=40, n_samples=250, size=2,
-                   zscale='linear'):
+def plot_objective(result, levels=10, n_points=40, n_samples=250, size=2, zscale='linear',
+        objective_name="Partial dependence", dim_names=None):
```

Comment:
Not a fan of `dim_names` as a name, how about just `dimensions`?

What is the use case for changing "Partial dependence" to something else?

---

formatting: can you indent this to line up with the opening bracket please?